osfoffline/views/preferences.py

```diff
@@ -147,8 +147,7 @@ def get_remote_top_level_nodes(self):
             if user:
                 user_nodes = []
                 url = api_url_for(USERS, related_type=NODES, user_id=user.osf_id)
-                # headers={'Authorization': 'Bearer {}'.format(user.oauth_token)}
-                headers = {'Cookie': 'osf_staging={}'.format(user.oauth_token)}
+                headers = {'Authorization': 'Bearer {}'.format(user.oauth_token)}
                 resp = requests.get(url, headers=headers).json()
                 logging.warning(resp)
```

Comment:
Followon comment: we should not need need to log the entire JSON payload to the console, much less as a warning.